dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx

```diff
@@ -218,6 +223,25 @@ cpdef list dpnp_meshgrid(xi, copy, sparse, indexing):
     return result
 
 
+cpdef dparray dpnp_ones(result_shape, result_dtype):
```

Comment:
Let's move the code of the function to separate place (maybe to `dpnp_algo`) and call it twice in both `dpnp_ones` and dpnp_zeros.

Reply:
this file included into dpnp_algo.pyx already. Since this function creates an array by some shapes - it looks like it is in the right place (other functions creates arrays from arrays).